satellite/audit/reverify_test.go

```diff
@@ -955,3 +960,192 @@ func TestReverifyExpired2(t *testing.T) {
 		require.Error(t, err)
 	})
 }
+
+// TestReverifySlowDownload checks that a node that times out while sending data to the
+// audit service gets put into containment mode
+func TestReverifySlowDownload(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 4, UplinkCount: 1,
+		Reconfigure: testplanet.Reconfigure{
+			NewStorageNodeDB: func(index int, db storagenode.DB, log *zap.Logger) (storagenode.DB, error) {
+				return testblobs.NewSlowDB(log.Named("slowdb"), db), nil
+			},
+			Satellite: func(log *zap.Logger, index int, config *satellite.Config) {
+				// These config values are chosen to force the slow node to time out without timing out on the three normal nodes
+				config.Audit.MinBytesPerSecond = 100 * memory.KiB
+				config.Audit.MinDownloadTimeout = 500 * time.Millisecond
+			},
+		},
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		satellite := planet.Satellites[0]
+		audits := satellite.Audit
+		queue := audits.Queue
+
+		audits.Worker.Loop.Pause()
+
+		ul := planet.Uplinks[0]
+		testData := testrand.Bytes(8 * memory.KiB)
+
+		err := ul.UploadWithConfig(ctx, satellite, &uplink.RSConfig{
+			MinThreshold:     2,
+			RepairThreshold:  2,
+			SuccessThreshold: 4,
+			MaxThreshold:     4,
+		}, "testbucket", "test/path", testData)
+		require.NoError(t, err)
+
+		audits.Chore.Loop.TriggerWait()
+		path, err := queue.Next()
+		require.NoError(t, err)
+
+		pointer, err := satellite.Metainfo.Service.Get(ctx, path)
+		require.NoError(t, err)
+
+		slowPiece := pointer.Remote.RemotePieces[0]
+		slowNode := slowPiece.NodeId
```

Comment:
nit/personal preference - could be cool to have a comment here saying "select one node to be slow" just to distinguish it from all the other setup happening.